.github/workflows/compute_slsa_source.yaml

```diff
@@ -0,0 +1,14 @@
+---
+name: SLSA Source
+on:
+  push:
+    branches: [ "main" ]
+permissions: {}
+
+jobs:
+  # Whenever new source is pushed recompute the slsa source information.
+  generate-provenance:
+    permissions:
+      contents: write # needed for storing the vsa in the repo.
+      id-token: write # meeded to mint yokens for signing
+    uses: slsa-framework/slsa-source-poc/.github/workflows/compute_slsa_source.yml@main
```

Comment:
we should use a release tag https://github.com/slsa-framework/slsa-source-poc/releases/tag/sourcetool%2Fv0.5.1